doc_user/UserManual.adoc

```diff
@@ -828,6 +829,409 @@ image:image251.png[success]
+
+====== 4.4.2.1.2 Insert Inventory Rows By Type And Usage
+
+To insert an inventory row for a set of medicals, useful for rotational inventories, click the *Add medicals by* button. This will open the *_Lot Information_* window.
+
+image:image255.png[Lot Information]
+
+Specify the Medical Type in the combo box (All = all medical types) and choose:
+
+* *All*: if all the pharmaceuticals for the selected Medical Type are needed.
+* *Only non-zero quantity*: if only the pharmaceuticals for the selected Medical Type with non-zero theoretical quantity are needed.
+* *With movement only*: if only the pharmaceuticals for the selected Medical Type which have been used at least once are needed.
+
+Then click on *[.underline]##O##k* button to insert all the pharmaceuticals that match the choice as the inventory(ies) row(s) in the *Inventory row table*.
+
+====== 4.4.2.1.3 Insert Inventory Row With New Lot
+
+If the pharmaceutical is already in the Inventory with its all possible lots, the system asks if to add it anyway:
+
+image:image255b.png[Add a medical]
+
+By clicking on the *[.underline]##Y##es* button, the pharmaceutical will be then added to the list with a *N* flag:
+
+image:image255c.png[New lot flag]
+
+This means that the lot has to be defined within the inventory and this can be achieved by selecting the inventory row and by clicking on the *[.underline]##L##ot* button:
+
+image:image255d.png[Define a lot]
+
+The procedure is pretty much the same as defined in chapter <<insert-stock-charging-movement,4.2.2.2 Insert Stock Charging Movement>>.
+
+Once the lot is defined, a real quantity greater than zero must be specified, otherwise validation will fail:
+
+image:image255e.png[Validation fail]
+
+===== 4.4.2.2 Edit Inventory ([.underline]##E##dit Inventory)
+
+To modify an inventory, select the inventory in the *_Inventory Browser_* window. Once the inventory has been highlighted, click on *[.underline]##E##dit* button to enter on the *_Edit Inventory_* window. Now the record is available for changes. This function allows editing all the data of the inventory including inventory(ies) row(s) of the inventory.
+
+NOTE: only inventories in DRAFT and VALIDATED states can be edited.
+
+image:image256.png[Edit Inventory]
+
+===== 4.4.2.3 Validate Inventory ([.underline]##V##alidate)
+
+During modification it is possible to validate the inventory by clicking on the *[.underline]##V##alidate* button. This may result in the following dialog:
+
+image:image271.png[Error]
+
+If the inventory has unsaved changes.
+
+image:image266.png[Validate Inventory]
+
+This Information window asks whether to check for any medicals on the stock or just the ones that are in the list?
```

Comment:
I would remove the question mark at the end of the line; a question isn't being asked here; the dialog is asking the question.

Reply:
Sure, thanks!